### DISCUSSION TITLE
attiny861a Bricked after writing Bootloader with 20Mhz External OSC

### DISCUSSION THREAD

**Discussion by daniele-v** [SpenceKonde/ATTinyCore#676] [Support / Q & A]
hi all, i am trying to program an attiny 861a chip with external oscillator configuration at 20Mhz. but when I try to program it the chip gets bricked and there is no more way to reprogram it. some idea ? for programming I'm using "ARDUINO as ISP" and as programming board a "Teensy4.0", everything works until I set the external oscillator to 20Mhz.
I have already stuck several chips and cannot understand the problem. thanks for any reply.

**Comment by SpenceKonde** [comment]
External Crystal or External Clock?  

If crystal is selected and there is neither clock nor crystal, the chip is bricked until you provide it with one or the other. 

If external clock selected, one must be provided, otherwise the chip will remain bricked. 

Add code to the sketch to output very high speed PWM (like 2-8 MHz) and connect to CLKI pin, and ten program it to

If external crystal is selected, and crystal is installed on the correct pins, it's not oscillating.. Likely bad connections or wrong value for loading caps. if it's breadboard, remove the loading caps entirely or use much lower value. There is a ton of parasitic capacitance in breadboard, often enough that adding the loading caps will push the total capacitance too high.

**Reply by daniele-v** [reply]
I found the "smd" crystal in wrong position, now everything works. thanks for the clarification.
